app/code/Magento/Ui/view/base/web/js/grid/columns/image-preview.js

```diff
@@ -0,0 +1,181 @@
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+define([
+    'underscore',
```

Comment:
Is `underscore` being used anywhere in the body of this module (`image-preview.js`)?

Reply:
removed